pkg/apic/apiservice_test.go

```diff
@@ -65,7 +65,13 @@ func TestCreateService(t *testing.T) {
 		},
 	})
 
-	apiSvc, err := client.PublishService(serviceBody)
+	// Test oas2 object
+	oas2Json, _ := os.Open("./testdata/petstore-swagger2.json") // OAS2
+	oas2Bytes, _ := ioutil.ReadAll(oas2Json)
+	cloneServiceBody := serviceBody
+	cloneServiceBody.Swagger = oas2Bytes
+
+	apiSvc, err := client.PublishService(cloneServiceBody)
```

Comment:
Don't see it anywhere, but I think if you open a file, you must close it, like this: defer oas2Json.Close().

Reply:
Thanks, I pushed it.